modules/commerce_bluesnap_currency/commerce_bluesnap_currency.module

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/**
+ * @file
+ * Provides Bluesnap currency resolver settings.
+ */
+
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implements hook_form_alter().
+ */
+function commerce_bluesnap_currency_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  // Alter commerce_currency_resolver_admin_conversion form and add
+  // bluesnap api settings.
+  if ($form_id == "commerce_currency_resolver_admin_conversion") {
```

Comment:
**CHANGE REQUEST**

*Readability*

Why don't you use `hook_form_FORM_ID_alter` instead?
If you keep it the way you have it, you need to avoid unnecessary indentation levels i.e.

```
if ($form_id !== "commerce_currency_resolver_admin_conversion") {
  return;
}

// Remaining code.
```

Reply:
I did not used `hook_form_FORM_ID_alter` as that would make the function name very long.
I have removed the unnecessary indentation